v1.0/sql/understanding-the-query-execution-plan.md

```diff
@@ -34,7 +34,9 @@ Currently, the `EXPLAIN` statement returns the following six columns: id, parent
 
 ### Introduction to task
 
-Currently, the calculation task of TiDB contains two different tasks: cop task and root task. The cop task refers to the computing task that is pushed to the KV side and executed distributedly. The root task refers to the computing task that is executed at a single point in TiDB. One of the goals of SQL optimization is to push the calculation down to the KV side as much as possible.
+Currently, there are two types of task execution: cop tasks and root tasks. A cop task refers to a computing task that is executed using TiKV Coprocessor. A root task refers to a computing task that is executed in TiDB.
```

Comment:
- I see the suggested version is "**in** TiKV coprocessor". Please check with @zz-jason which version is more accurate: `using TiKV Coprocessor` or `in TiKV Coprocessor`?
- Please also update this file in the master branch.

Reply:
I think it should either be _using the TiKV coprocessor_ (original) or _using/in TiKV's coprocessor_ (possessive).  Possessive struck me as more complex in language structure, so I wrote the original as _the_, but I am happy to use it if that is the preference.

Reply:
how about:
```suggestion
Currently, there are two types of task execution: cop tasks and root tasks. A cop task refers to a computing task that is executed in TiKV. A root task refers to a computing task that is executed in TiDB.
```

Reply:
I tend to agree with @morgo "Using the TiKV coprocessor" is a good choice here.

Reply:
@zz-jason your suggestion works too, but I like explaining coprocessor because then users can associate `cop` == coprocessor.  Without this hint they may not be sure why we called it `cop`.